jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -606,11 +578,7 @@ elif [[ -n "${IGN_PHYSICS_MAJOR_VERSION}" && ${IGN_PHYSICS_MAJOR_VERSION} -eq 2
 fi
 IGN_PHYSICS_DART_FROM_PKGS="true"
 
-IGN_PLUGIN_DEPENDENCIES="libignition-cmake1-dev"
-if [[ ${DISTRO} != 'xenial' ]]; then
-  IGN_PLUGIN_DEPENDENCIES="${IGN_PLUGIN_DEPENDENCIES} \\
-                           libignition-cmake2-dev"
-fi
+IGN_PLUGIN_DEPENDENCIES="libignition-cmake2-dev"
```

Comment:
If we're just removing the `xenial`, we should keep `cmake1` like was done for other instances above.

But `cmake1` has EOL'd already, so I think we should just remove that everywhere. As well as any other EOL versions. See this table as reference: https://github.com/ignitionrobotics/docs/blob/master/tools/versions.md

Reply:
https://github.com/ignition-tooling/release-tools/pull/478 removes EOL versions without touching the Xenial logic.